### ISSUE TITLE
occ_data function is giving me an error.

### ISSUE THREAD

**Issue by Gullgee** [ropensci/rgbif#431]
I have been using occ_data function and it was working fine. However, now it is giving me an error "Error in data.table::rbindlist(data, use.names = TRUE, fill = TRUE) : 
  Column 46 of item 1 is length 4 inconsistent with column 1 which is length 100. Only length-1 columns are recycled." 
Not sure how to resolve this issue

Here is the code I am using. 

for (i in 1:length(species_names)) {
   tryCatch({
    print(i)
    species <- species_names[i]
    #print.Date()
    sp_data <- rgbif::occ_data(scientificName=species,limit=100, hasCoordinate=TRUE)
    sp_data <- sp_data$data[,c("name", "decimalLatitude", "decimalLongitude", 
                               "year", "month", "day")]
    
    
  datalist[[i]] <- sp_data },# add it to your list
  error=function(e){cat("ERROR :",conditionMessage(e), "\n")})
  }
> <summary><strong>Session Info</strong></summary>

```r

```
</details>

**Comment by AMBarbosa** [comment]
Dear colleagues,
I'm getting a similar error for a simple query (see below). I've used `rgbif` regularly and this did not happen until now. Something to do with `data.table::rbindlist`, maybe caused by an update to the function or to the GBIF data structure?

```
library(rgbif)
```
> Warning message:
> package ‘rgbif’ was built under R version 4.0.2 

```
occ_data(scientificName = "Gorilla beringei")
```
> Error in data.table::rbindlist(data, use.names = TRUE, fill = TRUE) : 
>   Column 48 of item 1 is length 4 inconsistent with column 1 which is length 300. Only length-1 columns are recycled.

```
traceback()
```
> 4: data.table::rbindlist(data, use.names = TRUE, fill = TRUE)
> 3: data.table::setDF(data.table::rbindlist(data, use.names = TRUE, 
>        fill = TRUE))
> 2: .get_occ_data(curlopts = curlopts)
> 1: occ_data(scientificName = "Gorilla beringei")

```
sessionInfo()
```

> R version 4.0.0 (2020-04-24)
> Platform: x86_64-w64-mingw32/x64 (64-bit)
> Running under: Windows 10 x64 (build 18363)
> 
> Matrix products: default
> 
> locale:
> [1] LC_COLLATE=Portuguese_Portugal.1252 
> [2] LC_CTYPE=Portuguese_Portugal.1252   
> [3] LC_MONETARY=Portuguese_Portugal.1252
> [4] LC_NUMERIC=C                        
> [5] LC_TIME=Portuguese_Portugal.1252    
> 
> attached base packages:
> [1] stats     graphics  grDevices utils     datasets  methods  
> [7] base     
> 
> other attached packages:
> [1] rgbif_3.2.0 sp_1.4-1   
> 
> loaded via a namespace (and not attached):
>  [1] Rcpp_1.0.4.6      pillar_1.4.4      compiler_4.0.0   
>  [4] plyr_1.8.6        tools_4.0.0       uuid_0.1-4       
>  [7] jsonlite_1.6.1    lifecycle_0.2.0   tibble_3.0.1     
> [10] gtable_0.3.0      lattice_0.20-41   pkgconfig_2.0.3  
> [13] rlang_0.4.6       rstudioapi_0.11   curl_4.3         
> [16] crul_0.9.0        dplyr_0.8.5       httr_1.4.1       
> [19] stringr_1.4.0     xml2_1.3.2        vctrs_0.3.0      
> [22] rgeos_0.5-3       triebeard_0.3.0   grid_4.0.0       
> [25] tidyselect_1.1.0  httpcode_0.3.0    glue_1.4.0       
> [28] data.table_1.12.8 geoaxe_0.1.0      R6_2.4.1         
> [31] oai_0.3.0         conditionz_0.1.0  ggplot2_3.3.0    
> [34] purrr_0.3.4       magrittr_1.5      whisker_0.4      
> [37] urltools_1.7.3    scales_1.1.1      ellipsis_0.3.0   
> [40] assertthat_0.2.1  colorspace_1.4-1  stringi_1.4.6    
> [43] lazyeval_0.2.2    munsell_0.5.0     crayon_1.3.4

**Comment by sckott** [comment]
thanks for the issue!  this should already be fixed, via #427 

install like `remotes::install_github("ropensci/rgbif")` or `install.packages("rgbif", repos="https://dev.ropensci.org")`
